### PR TITLE
Addressed Laura's final content review comments

### PR DESCRIPTION
**Pull request by ReeceNana**
Draft-Guide-Jakarta-Websocket

Title and abstract

- [x] This feels very technical and dry as a title and abstract. It doesn't really give a sense of why a developer might want to use websocket. Maybe something more like "Bidirectional communication between services using Jakarta WebSocket". [@yeekangc](https://github.com/yeekangc) Any better suggestions?
- [x] Also, the abstract needs to provide more info than just a re-ordering of the words in the title. Otherwise, there's no point having the abstract. eg "Send and receive messages between services without closing the connection." (that's not very good but something like that - needs to expand on what the title says to clarify it)

What you'll learn
- [x] Similarly, I think this opening para needs beefing up. It starts by giving a technical explanation then compares WebSocket to SSE as if the reader is likely to know SSE but not WebSocket. That may be true for some (and I think having the comparison made explicit here is a good thing) but it won't be for all, so I think we need a little more explanation before the SSE comparison. I think it's worth mentioning that it's because the connection isn't closed so that the client can continue to send and receive messages with the server, without having to poll the server to receive any replies.
- [x] We shouldn't be linking to the spec - they shouldn't have to read the spec to use or learn about the product capabilities. We don't want to lose them to wading through a spec looking for useful user information. We're the target audience for specs, not developers learning to use Liberty. If they care about specs, they can google for it. There's a load of Javadoc here: https://openliberty.io/docs/latest/reference/javadoc/liberty-jakartaee9.1-javadoc.html?package=jakarta/websocket/package-frame.html&class=overview-summary.html I've also raised a [doc issue](https://github.com/OpenLiberty/docs/issues/6071) to create doc on the communcation options available in Liberty and when each should be used. That probably won't be written in time for this guide (in future, when you spot gaps in the OL docs, just raise an issue - we can always cancel it if it's not necessary). /cc [@dmuelle](https://github.com/dmuelle) " Jakarta WebSocket API" could be a link to the Javadoc, and the current link to the spec should be removed for now and, in future (create an issue for a future update), replaced with a link to the doc topic in the doc issue.
- [x] The SSE reference should probably link to our SSE guide (I think we have one?).
- [x] "First, each client makes a connection to a Jakarta WebSocket server by using HTTP." I would rephrase to "First, each client makes an HTTP connection to a Jakarta WebSocket server." ("by using" sounds really odd here and emphasises the connection protocol as if using HTTP is the key thing for the developer to do to. The important thing is that they make the connection (WebSocket just uses HTTP). Though, actually, doesn't WebSocket use TCP and not HTTP anyway? I don't understand the relevance of mentioning HTTP here...
- [x] "every 10 seconds", not "in every 10 seconds".
- [x] "Then, the system service broadcasts messages to clients." this isn't really a complete sentence. How about "The scheduler pushes messages to the system service every 10 seconds, then the system service broadcasts the messages to any connected clients."

Creating the WebSocket server service

- [x] "When you run Open Liberty in development mode, known as dev mode," should just say "When you run Open Liberty in dev mode," There's no such thing as "development mode" in this context.
- [x] We shouldn't be saying "application server" any more (this may also need an issue against older guides). We use "runtime" as "application server" is old-fashioned.

Creating the client service

- [x] As for the server service (re dev mode and runtime).
- [x] We shouldn't be linking to the spec from the guides - they shouldn't have to read the spec to use the product. We could link to the Javadoc (as hosted in the OL docs)? eg https://openliberty.io/docs/latest/reference/javadoc/liberty-jakartaee9.1-javadoc.html?package=jakarta/ejb/package-frame.html&class=jakarta/ejb/EnterpriseBean.html though that link doesn't mention the Timer service so it's not that helpful - maybe this? https://openliberty.io/docs/latest/reference/javadoc/liberty-jakartaee9.1-javadoc.html?package=jakarta/ejb/package-frame.html&class=jakarta/ejb/TimerService.html
- [x] Are there any other places we should be linking to the OL docs and/or the Javadoc?

Testing the application

- [ ] "When you are done checking out the services, exit dev mode by pressing CTRL+C in the command-line sessions where you ran the system and client services, or by typing q and then pressing the enter/return key. Alternatively, you can run the liberty:stop goal from the start directory in another command-line session for the system and client services:" That's way too many different options for a guide. Just pick the best/easiest one and tell them to do that. At the very least, don't tell them both CTRL+C and 'q' - just pick one.

### CODE REVIEW

**Review by gkwan-ibm — COMMENTED** [review]
hi @ReeceNana 

line 37, "...the clients.link:https:..." -> "...the clients. link:https:..." 
- need a space

**Comment by gkwan-ibm** [comment]
The issue https://github.com/OpenLiberty/guides-common/issues/866 will address the "Testing the application" comment for all guides

**Review by lauracowen — APPROVED** [review]
Looks great, thanks @ReeceNana . I especially like the new opening paragraph and the way you've contrasted WebSocket with SSE - I think it's now clear why SSE is mentioned and the key differences between the two (why you'd choose one or the other).

**Review by gkwan-ibm — APPROVED** [review]
